src/components/Popup/Popup.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import "./popup.css";
+import TextareaAutosize from 'react-textarea-autosize';
+
+function Popup({ trigger, setTrigger, card, updateCard }) {
+  const [name, setName] = useState(card.name);
+  const [isDone, setIsDone] = useState(card.isDone);
```

Comment:
Why didn't you aggregate both fields in object?

Reply:
done